sonoff/my_user_config.h

```diff
@@ -269,6 +269,10 @@
 // -- Internal Analog input -----------------------
 #define USE_ADC_VCC                              // Display Vcc in Power status. Disable for use as Analog input on selected devices
 
+// -- BOOT Delay for drivers ----------------------
+
+#define DRIVER_BOOT_DELAY       1                // Number of milliseconds to retard driver cycles during boot-up time to reduce overall CPU load whilst Wifi is connecting
```

Comment:
What about adding this define in sonoff.h instead?

Reply:
Yeah. That's better. Keep it from "normal" users.

Reply:
For me, it does not matter where it is but it would be easier to configure it for pre-compiled binaries if it's in my_user_config.h - But let's see... may not even include it given the results below.

Reply:
Yeah, didn't think of that... will be safer in sonoff.h ;)

Reply:
Moved